test-suite-linker/src/main/scala/org/scalajs/bootstrap/TestSuiteLinker.scala

```diff
@@ -22,7 +22,11 @@ object QuickLinker {
 
     val linker = StandardLinker(config)
 
-    val moduleInitializers = build.TestSuiteLinkerOptions.moduleInitializers
+    val moduleInitializers = {
+      build.TestSuiteLinkerOptions.moduleInitializers :+
+      // Copied from org.scalajs.testadapter.TestAdapaterInitializer.
+      ModuleInitializer.mainMethod("org.scalajs.testinterface.Bridge", "start")
```

Comment:
So, this is obviously not very nice :S The other option we have is to start cross compiling parts of the `test-adapter` package (although it will never make sense to use it in JS, since by definition it is a JS/JVM bridge).

WDYT we should do here?

Reply:
The alternative is worse, IMO. I suggest we keep it this way, with the hard-coded copy of `TestAdapterInitializer` in `TestSuiteLinker`.